### PR TITLE
Support new kubeversion for curated package e2e tests

### PR DESCRIPTION
**Pull request by vincentni**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/e2e/nutanix_test.go [hunk 1]
```diff
@@ -14,214 +14,104 @@ import (
...
+	case v1alpha1.Kube123:
+		return framework.WithUbuntu123Nutanix()
+	case v1alpha1.Kube124:
+		return framework.WithUbuntu124Nutanix()
+	case v1alpha1.Kube125:
+		return framework.WithUbuntu125Nutanix()
+	case v1alpha1.Kube126:
+		return framework.WithUbuntu126Nutanix()
+	default:
+		return framework.WithUbuntu127Nutanix()
```
Comment (vivek-koppuru) [inline test/e2e/nutanix_test.go:28]: Is it possible to have a different version? If we have 122, then will 127 work?

File: test/e2e/metallb.go [hunk 2]
```diff
@@ -28,10 +27,8 @@ type MetalLBSuite struct {
 	provider          framework.Provider
 }
 
-var versionsToTest = []v1alpha1.KubernetesVersion{v1alpha1.Kube124, v1alpha1.Kube125}
-
 func RunMetalLBDockerTests(t *testing.T) {
-	for _, v := range versionsToTest {
+	for _, v := range KubeVersions {
```
[the remark this answers is not in the document]
Comment (vincentni) [reply test/e2e/metallb.go:31]: yes, it goes through all supported versions and tests each of them.

File: test/e2e/nutanix_test.go (hunk 1 above)
Comment (vincentni) [reply test/e2e/nutanix_test.go:28]: The idea here is the default case is left for the most update-to-date version, and all the previous versions have to be explicitly listed.

**Review by vivek-koppuru — APPROVED** [review]

**Pull request merged**